.requirements

```diff
@@ -10,3 +10,10 @@ LIBYAML_VERSION=0.2.5
 KONG_GO_PLUGINSERVER_VERSION=v0.6.1
 KONG_BUILD_TOOLS_VERSION=4.40.1
 KONG_NGINX_MODULE_BRANCH=0.2.1
```

Comment:
It seems pre-bazel, on the `2.8.4` tag, we were:
```
KONG_PACKAGE_NAME=kong
KONG_CONFLICTS=kong-enterprise-edition
KONG_LICENSE="ASL 2.0"

RESTY_VERSION=1.19.9.1
RESTY_LUAROCKS_VERSION=3.8.0
RESTY_OPENSSL_VERSION=1.1.1o
RESTY_PCRE_VERSION=8.45
LIBYAML_VERSION=0.2.5
KONG_GO_PLUGINSERVER_VERSION=v0.6.1
KONG_BUILD_TOOLS_VERSION=4.40.1
KONG_NGINX_MODULE_BRANCH=0.2.1
```

~ https://github.com/Kong/kong/blob/2.8.4/.requirements

Reply:
Unless we can't avoid it, if possible, we probably don't want to change any of these versions.

Reply:
https://github.com/Kong/lua-kong-nginx-module/commit/f52a34c17af4543245ff79e3227b9d514b8cfa5c
https://github.com/Kong/kong/commit/edf183546ca23dc4b4e771b7325b0f483286ad09

It seems that some new features from version 0.2.2 of lua-kong-nginx-module are being used.

Reply:
We downgrade these dependencies' versions to try to keep them consistent with version 2.8.4.
https://github.com/Kong/kong/pull/13282